apps/src/code-studio/components/SmallFooter.jsx

```diff
@@ -156,16 +155,25 @@ export default class SmallFooter extends React.Component {
   };
 
   render() {
+
     const styles = {
       smallFooter: {
         fontSize: this.props.fontSize
       },
-      base: _.assign({}, this.props.baseStyle, {
+      base: {
+        paddingBottom: 3,
+        paddingTop: 3,
+        // subtract top/bottom padding from row height
+        height: this.props.rowHeight ? this.props.rowHeight - 6 : undefined
+      },
+      baseRestrictWidth: {
+        maxWidth: '100%',
+        boxSizing: 'border-box',
         paddingBottom: 3,
         paddingTop: 3,
         // subtract top/bottom padding from row height
         height: this.props.rowHeight ? this.props.rowHeight - 6 : undefined
-      }),
+      },
```

Comment:
We have a bunch of duplicated styling here. You could have `baseRestrictWidth` just specify maxWidth/boxSizing, and then on line 213 do:
```
const baseStyle = {
   ...styles.base,
   ...(this.props.restrictWidth && styles.baseRestrictWidth),
```
I don't feel particularly strongly about this.

Reply:
That was bugging me too.  We don't seem to have settled on a strategy for this yet, and we've had a lot of examples when we start facing RTL and responsive variants of existing styles.  I wonder if we could develop a naming convention for "additional" styling?